ui/src/api/src/__mocks__/ApiClient.js

```diff
@@ -0,0 +1,35 @@
+import { getFacetsList } from "../../../../tests/mockutils";
```

Comment:
I didn't realize you wrote these files, since git hid diffs by default. Please modify .gitattributes so diffs from `__mocks__` directory are shown by default.

Reply:
Done